### PR TITLE
Updates for Selenium and web drivers

### PR DESCRIPTION
**Pull request by dbudwin**
Updates to the latest version of Selenium and web drivers.

Selenium updated from 2.53.1 -> 3.4.0
Web drivers updated from 2.24 -> 2.29

### CODE REVIEW

**Comment by Modelizer** [comment]
Have you tested all the APIs with selenium 3.0 version? Also it would be great if you submit test cases for it.

File: README.md [hunk 1]
```diff
@@ -163,3 +163,4 @@ Just do it. You are welcome :)
 | Contributors           | Twitter   | Ask for Help | Contact / Hire  | Site            |
 |------------------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|-----------------|-----------------|
 | [Mohammed Mudasir](https://github.com/Modelizer) (Creator) | @[md_mudasir](https://twitter.com/md_mudasir) | [![Get help on Codementor](https://cdn.codementor.io/badges/get_help_github.svg)](https://www.codementor.io/modelizer) | hello@mudasir.me | [http://mudasir.me](http://mudasir.me/) |
+| [Drew Budwin](https://github.com/dbudwin) (Contributor) | - | - | [Linkedin](https://www.linkedin.com/in/dbudwin/) | [https://budw.in](https://budw.in/) |
```
Comment (Modelizer) [inline README.md:166]: I encourage you to write as a maintainer and help me out maintaining this package. It will help me alot. Thanks.

**Comment by dbudwin** [comment]
No I didn't test all the APIs.  I'm somewhat new to Selenium which is why I found this package yesterday.  I was experiencing issues with Selenium and almost every answer I came across told me to upgrade the version of Selenium I was using.  I really like this package because it takes away the ability to have to install packages.  However, locally on my machine, upgrading Selenium just traded one error for another.  I can't even get the simple test case provided in the README to pass.  If I run the example in the README I get the following error:

`Error connection[28] to http://localhost:4444/wd/hub/session/c9423014-8b91-4fe0-88f8-547d1875e7b1/cookie: Operation timed out after 60003 milliseconds with 0 bytes received`

I looked through the PR history a bit to see if I could find an example of how you prefer to have it done but I didn't find any.  How do you propose I move forward with this PR.

**Comment by Modelizer** [comment]
Please test with APIs so that we can proceed further by merging this.

**Comment by Modelizer** [comment]
I have upgraded selenium  to 3.11.0 and chrome drive to 2.35. Please upgrade to 2.0 version of this package if still you are using it.

**Pull request closed without merging**